.github/workflows/db-migration-test.yml

```
@@ -0,0 +1,123 @@
+          MYSQL_ROOT_PASSWORD: root
+          MYSQL_DATABASE: matchday_test
+        ports:
+          - 3306:3306
+        options: >-
+          --health-cmd="mysqladmin ping"
+          --health-interval=10s
+          --health-timeout=5s
+          --health-retries=3
+
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)